index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google Default webpage</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet"  href="css/all.css">
+</head>
+<body>
+    <ul>
+        <li a href="#Gmail">Gmail</li> 
+        <li a href="#Images">Images</li> 
+        <i class="fas fa-th"></i> 
+    <nav><li class="Sign In">Sign In</li></nav>
+    </ul>
+
+    <div>
+        <Img src="Go.png">
+           <p><i class="fas fa-microphone"></i></p>
+    </div>
+    
+    <div class="flex-container">
+        <div>Google Search</div>
+        <div>I'm Feeling Lucky</div>
+      </div>
```

Comment:
After giving this **div** a class attribute, i think what should follow next should be an **h4** heading and not another **div** elements